staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_filter_test.go

```diff
@@ -115,18 +115,18 @@ func TestQueueWaitTimeLatencyTracker(t *testing.T) {
 		QueueSetFactory:        fqs.NewQueueSetFactory(clk),
 	})
 
-	stopCh := make(chan struct{})
-	defer close(stopCh)
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
In unit tests use `_, ctx := ktesting.NewTestContext(t)` to get per-test output.